src/treq/_utils.py

```diff
@@ -45,3 +48,50 @@ def default_pool(reactor, pool, persistent):
         set_global_pool(HTTPConnectionPool(reactor, persistent=True))
 
     return get_global_pool()
+
+
+def warn(*args, **kwargs):
```

Comment:
Do we want to use `functools.wraps` or `wrapt` or...  If we do that can we have extra args?  I certainly have had places I should have used them but have never really learned them.